plugin/core/views.py

```diff
@@ -164,9 +168,9 @@ def entire_content_range(view: sublime.View) -> Range:
     return region_to_range(view, entire_content_region(view))
 
 
-def text_document_item(view: sublime.View, language_id: str) -> Dict[str, Any]:
+def text_document_item(view: sublime.View, language_id: str, file_name: Optional[str] = None) -> Dict[str, Any]:
     return {
-        "uri": uri_from_view(view),
+        "uri": filename_to_uri(file_name) if file_name is not None else uri_from_view(view),
```

Comment:
Why do we accept empty `file_name` for the string case? `uri_from_view` does not do that.

Reply:
Turns out this change is not needed after all.